projects/aws/eks-a-admin-image/ova/vsphere.pkrvars.hcl

```diff
@@ -0,0 +1,12 @@
+
+# credential vars get passed via env var in the makefile
+#vsphere_password 
+#vsphere_username
+
+vsphere_insecure_connection  = false
+
+vsphere_endpoint        = "vcenter.sddc-44-239-186-141.vmwarevmc.com"
+vsphere_datacenter      = "SDDC-Datacenter"
+vsphere_datastore       = "WorkloadDatastore"
+vsphere_cluster         = "Cluster-1"
+vsphere_network         = "/SDDC-Datacenter/network/sddc-cgw-network-1"
```

Comment:
do we want to have this committed in the repo?
We could make it dynamic and just pass the right values in the codebuild job

Reply:
Yeah I can move these out to env vars.